saleor/demo/settings.py

```diff
@@ -28,7 +28,8 @@
 
 PLUGINS += ["saleor.plugins.anonymize.plugin.AnonymizePlugin"]
 
-GRAPHENE["MIDDLEWARE"].insert(0, "saleor.graphql.middleware.ReadOnlyMiddleware")  # type: ignore
+if "MIDDLEWARE" in GRAPHENE.keys():
+    GRAPHENE["MIDDLEWARE"].insert(0, "saleor.graphql.middleware.ReadOnlyMiddleware")  # type: ignore
```

Comment:
It should be always enabled on demo
```suggestion
GRAPHENE.setdefault("MIDDLEWARE", []).insert(0, "saleor.graphql.middleware.ReadOnlyMiddleware")  # type: ignore
```

Reply:
fixed